packages/workspace/src/browser/i18n.spec.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import fs = require('fs');
+import "mocha";
+import * as chai from "chai";
+const Globalize = require("globalize");
+
+const expect = chai.expect;
+
+beforeEach(() => {
+});
+
+// describe('internationalization', () => {
+
+describe('Some menu labels', () => {
+    // Load the json file for each string in different languages
+    loadMessagesfiles();
+    Globalize.load(
+        // Core
+        require("cldr-data/supplemental/likelySubtags"),
+        require("cldr-data/supplemental/plurals")
+    );
+
+    it('Test Internationalize string in English', () => {
```

Comment:
Should we not test with MessageTranslation ?

Reply:
Done, testing with MessageTranslations